pkg/exposer/generic_restore.go

```diff
@@ -165,7 +165,12 @@ func (e *genericRestoreExposer) CleanUp(ctx context.Context, ownerObject corev1.
 	restorePVCName := ownerObject.Name
 
 	kube.DeletePodIfAny(ctx, e.kubeClient.CoreV1(), restorePodName, ownerObject.Namespace, e.log)
-	kube.DeletePVCIfAny(ctx, e.kubeClient.CoreV1(), restorePVCName, ownerObject.Namespace, e.log)
+	pvc, err := e.kubeClient.CoreV1().PersistentVolumeClaims(ownerObject.Namespace).Get(ctx, restorePVCName, metav1.GetOptions{})
```

Comment:
Let's put the PVC existence detection and get PV action into `DeletePVAndPVCIfAny` function, so that we don't need to change every places calling `DeletePVAndPVCIfAny`